packages/formatron-react/src/form/fluentui/Enum.tsx

```diff
@@ -20,16 +22,30 @@ const enumValuesToDropdownOptions = (
     text: value.display,
     disabled: value.disabled
   }));
+const enumValuesToChoiceGroupOptions = (
+  values: EnumValueType[]
+): IChoiceGroupOption[] =>
+  values.map((value) => ({
+    key: value.alias,
+    text: value.display,
+    disabled: value.disabled
+  }));
 
 export interface EnumProps extends CommonFieldProps {
   field: FieldObjectDataType<EnumType>;
+  /**
+   * Displays the enum values as radio buttons instead of a dropdown.
+   */
+  radio?: boolean;
   dropDownProps?: Partial<IDropdownProps>;
 }
 
 export const Enum: React.FC<EnumProps> = ({
   field,
   dropDownProps = {},
-  disabled = false
+  disabled = false,
+  radio = false,
```

Comment:
Very nit-picky, but can we follow the boolean naming convention and use `isRadioGroup` here?